index.html

```diff
@@ -3192,7 +3209,7 @@ <h3><dfn>Writing content</dfn></h3>
               <li>
                 Set the |ndef|'s <a>PAYLOAD field</a> to the result of running
                 the <a>create NDEF message</a> given |record|'s <a>data</a>
```

Comment:
`data` here is `any` while the "create NDEF message" record expects `NDEFMessageSource`. It feels like we need to acknowledge this but maybe not because those steps do try to match on the parameter type and have a default case which will throw a TypeError.

Reply:
I believe it's okay to leave it as it. As you said, an `unmatched type` will throw an error in the "create NDEF message" algorithm